src/item_pocket.h

```diff
@@ -381,6 +385,17 @@ class item_pocket
 
         favorite_settings settings;
 
+        // Pocket presets functions
+        void serialize_presets( JsonOut &json ) const;
+        void deserialize_presets( const JsonArray &ja );
+        void load_presets();
+        void add_preset( const item_pocket::favorite_settings preset );
+        void save_presets();
+        std::vector<item_pocket::favorite_settings>::iterator find_preset( const std::string s );
+        bool has_preset( const std::string s );
```

Comment:
```suggestion
        std::vector<item_pocket::favorite_settings>::iterator find_preset( const std::string &s );
        bool has_preset( const std::string &s );
```
Use reference to avoid a copy.